index.css

```diff
@@ -1,246 +1,130 @@
+    left: 0;
+    top: 0;
+    background-color: #fff;
+    z-index: -1;
+    border-bottom-left-radius: 80px;
   }
 
-  /* Shapes */
+  /* SHAPES */
   #shapes-background {
-   position: absolute;
-   width: 462px;
-   height: 587.15px;
-   left: -34px;
-   top: 104px;
-  }
-
-  #shapes-background img {
-   position: absolute;
-  }
-
-  #shapes-background .rectangle-4 {
-   top: 14px;
-   left: 328px;
-  }
-
-  #shapes-background .ellipse-3 {
-   top: 48.24px;
-   left: 357px;
-  }
-
-  #shapes-background .ellipse-8 {
-   top: 6px;
-   left: 214px;
-  }
-
-  #shapes-background .rectangle-6 {
-   right: 419.27px;
-  }
-
-  #shapes-background .ellipse-11 {
-   top: 24.64px;
-   right: 404.4px;
-  }
-
-  #shapes-background .rectangle-7 {
-   top: 44px;
-   right: 277.68px;
-  }
-
-  #shapes-background .ellipse-5 {
-   top: 51.81px;
-   left: 159px;
-  }
-
-  #shapes-background .ellipse-14 {
-   left: 332px;
-   top: 211px;
-  }
-
-  #shapes-background .ellipse-12 {
-   top: 429px;
-   left: 351px;
-  }
-
-  #shapes-background .ellipse-13 {
-   top: 440px;
-   left: 338px;
-  }
-  
-  #shapes-background .ellipse-15 {
-   top: 541px;
-   left: 87px;
-  }
-
-  #shapes-background .ellipse-4 {
-   top: 553.1px;
-   left: 271px;
-  }
-
-  #shapes-background .rectangle-5 {
-   top: 549px;
-   left: 221px;
+    position: absolute;
+    width: 100%;
+    height: 587.15px;
+    top: 104px;
+    background-image: url("./assets/images/shapes-background.png");
   }
 
   /* HEADLINE */
   #headline {
-   display: flex;
-   flex-direction: column;
-   align-items: flex-start;
-   padding: 114px 24px;
-   gap: 12px;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 114px 24px;
+    gap: 12px;
   }
 
   #headline .headline-title {
-   color: var(--title-color);
-   font-style: normal;
-   font-weight: 700;
-   font-size: 40px;
-   line-height: 53px;
-   display: flex;
-   align-items: center;
-   margin-bottom: 0px;
+    color: #172b4d;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 53px;
+    align-items: center;
+    width: 300px;
   }
 
   #headline .headline-text {
-   color: var(--text-color);
-   font-style: normal;
-   font-weight: 400;
-   font-size: 16px;
-   line-height: 24px;
-   margin-top: 0px;
-   margin-bottom: 0px;
+    color: #344563;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    margin-top: 0;
+    margin-bottom: 0;
   }
 
   #headline .headline-supporting-text {
-   display: flex;
-   flex-direction: row;
-   align-items: center;
-   padding: 0px;
-   gap: 8px;
-   width: 117px;
-   height: 24px;
-  }
-
-  .headline-supporting-text p {
-   color: var(--secondary-color);
-   font-style: normal;
-   font-weight: 500;
-   font-size: 16px;
-   line-height: 24px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 0;
+    gap: 8px;
+    width: 117px;
+    height: 24px;
```

Comment:
- []Optional] Good Job 👍 👍  I think it is better to use class only for reusability, you can create such class`.primary-title` and define it for later use like `aboutme` section. 😉😉

Reply:
Thanks for the advice!